### ISSUE TITLE
Can't install together with other pubtools.* modules

### ISSUE THREAD

**Issue by rohanpm** [release-engineering/pubtools-pulplib#31]
#### Summary

This project currently ships modules under the pubtools.pulplib hierarchy.

It is not currently possible to install and use this together with another egg which also wants to install modules somewhere under pubtools.*, because we haven't packaged pubtools as a "namespace package".

#### Issue Type
- Bug Report

#### Steps to reproduce

- Install this project (in a virtualenv)
- Install https://github.com/release-engineering/pubtools-pulp to same virtualenv
- Attempt to import & use modules from both projects

#### Actual results

Usage of at least one of the projects will fail with ModuleNotFoundError.

#### Expected results

Modules from both projects can be imported and used successfully.

#### OS / Environment

all

#### Additional Information

The appropriate solution for us seems to be: https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages

In summary, pubtools/\_\_init\_\_.py in both this project and every other sharing the pubtools namespace ought to contain:

```
__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```
